imports/server/accounts.ts

```diff
@@ -52,26 +70,79 @@ const summaryFromLoginInfo = function (info: LoginInfo) {
   }
 };
 
-Accounts.registerLoginHandler((options: LoginOptions) => {
+Accounts.registerLoginHandler(async (options: LoginOptions) => {
```

Comment:
Hmm. I think that the current implementation is technically safe, although mostly by accident, but we should do some `check`-style type verification here before passing client-provided input as a query parameter. Otherwise someone could do `huntInvitationCode: {"$ne": null}` or something.

(I think the current implementation is safe because `Google.retrieveCredential` validates that `key` is a string before passing it to the database, and `key` looks to mostly be a random 40-byte-ish string that I think is unguessable by an attacker)

Reply:
Looks like you added an initial `check` here - I extended it with the new parameters added in this PR. Thanks!